app/assets/stylesheets/content/_forms.sass

```diff
@@ -535,6 +535,11 @@ fieldset.form--fieldset
   & > [data-tooltip], & > [class^="tooltip--"]
     padding-top: 0.6rem
 
+.form--field
+  & .-break-words > .form--label
+    text-overflow: ellipsis
+    overflow: hidden
```

Comment:
> And it is probably the better approach to always apply ellipsis to form labels

I am not sure in whcih places this would apply, but be careful because we should not go back and forth here.

before I paid attention someone added elipsis to many places, which resulted in all texts being unreadable and finally to this WP: https://community.openproject.org/work_packages/19428 (check the screenshots) and this PR: https://github.com/opf/openproject/pull/2873.

I just want to avoid that we go back and forth between "I can read no texts" and "my texts overlap my inputs".